docs/develop-resources/guides/4-add-controller.md

```diff
@@ -45,6 +45,15 @@ hack/compare-mock fixtures/<your_resource_test>
 KCC_USE_DIRECT_RECONCILERS=<YOUR KIND> hack/compare-mock fixtures/<your_resource_test>
 ```
 
+### Turn on SciFi Controller
+
+To fully turn on the SciFi controller, add your resource to [KCC_USE_DIRECT_RECONCILERS](https://github.com/xiaoweim/k8s-config-connector/blob/master/dev/tasks/run-e2e#L27) and make sure it passes all tests and there is a decent test coverage over customer use cases. Usually we do not turn it on for beta resources right away.
+ 
+
+ * Note: some resources are used as dependency resources in other test scenarios, to only test the direct controller with your resource test, you can use this annotation `alpha.cnrm.cloud.google.com/reconciler: "direct"` in the test yamls instead of using the `KCC_USE_DIRECT_RECONCILERS` flag at CLI.
```

Comment:
Let's not introduce these two options right now. It could cause some confusions

Reply:
Sure, my point is actually focused on that maybe we should not switch the resource to using direct controller right away after migration, but at the same time also test the direct controller in our e2e tests. That is why I mentioned `alpha.cnrm.cloud.google.com/reconciler: "direct"` annotation. I am not sure if there's a better way to emphasize on this point. But I see there could be some confusion rising from this.